src/main/java/org/candlepin/subscriptions/tally/InventoryAccountUsageCollector.java

```diff
@@ -196,4 +203,7 @@ else if (facts.isVirtual() && !StringUtils.isEmpty(facts.getHypervisorUuid())) {
         return calcsByAccount;
     }
 
+    protected Counter getTotalHosts() {
+        return totalHosts;
+    }
```

Comment:
Rather than expose the `Counter` as `protected`, consider this instead: `@Autowire` the MeterRegistry in the test, and then call `registry.counter("rhsm-subscriptions.capacity.records_total")` - this will return the same counter instance.

Reply:
Okay, good.  I will rewrite this.